docs/getting_started_windows_vulkan.md

```diff
@@ -0,0 +1,130 @@
+# Getting Started on Windows with Vulkan
+
+[Vulkan](https://www.khronos.org/vulkan/) is a new generation graphics and
+compute API that provides high-efficiency, cross-platform access to modern GPUs
+used in a wide variety of devices from PCs and consoles to mobile phones and
+embedded platforms.
+
+IREE includes a Vulkan/[SPIR-V](https://www.khronos.org/spir/) HAL backend
+designed for executing advanced ML models in a deeply pipelined and tightly
+integrated fashion on accelerators like GPUs.
+
+This guide will walk you through using IREE's compiler and runtime Vulkan
+components.
+
+## Prerequisites
+
+You should already have IREE cloned and building on your Windows machine. See
+the [Getting Started on Windows with CMake](getting_started_windows_cmake.md)
+guide for instructions.
+
+You must have a physical GPU with drivers supporting Vulkan. We support using
+[SwiftShader](https://swiftshader.googlesource.com/SwiftShader/) (a high
+performance CPU-based implementation of Vulkan) on platforms where the
+[Vulkan-ExtensionLayer](https://github.com/KhronosGroup/Vulkan-ExtensionLayer)
+project builds, but it does not currently
+[build on Windows](https://github.com/KhronosGroup/Vulkan-ExtensionLayer/issues/16).
+
+Vulkan API version > 1.2 is recommended, though older versions with support for
+the `VK_KHR_timeline_semaphore` extension may also work.
+
+## Vulkan Setup
+
+### Background
+
+Vulkan applications interface with Vulkan "drivers", "layers", and "extensions"
+through the Vulkan loader. See LunarG's
+[Architecture of the Vulkan Loader Interfaces](https://vulkan.lunarg.com/doc/view/latest/windows/loader_and_layer_interface.html) page for more information.
+
+### Quick Start
+
+The
+[dynamic_symbols_test](https://github.com/google/iree/blob/master/iree/hal/vulkan/dynamic_symbols_test.cc)
+checks if the Vulkan loader and a valid ICD are accessible.
+
+To configure CMake then build and run the test:
+
+```shell
+$ cmake -B build\ .
```

Comment:
This can be omitted or turned into a comment like `// Assumes cmake -B build\ .`?